library/core/src/unicode/rt.rs

```diff
@@ -126,3 +128,30 @@ unsafe fn skip_search<const SOR: usize, const OFFSETS: usize>(
     }
     offset_idx % 2 == 1
 }
+
+#[inline(always)]
+pub(super) fn case_conversion(
```

Comment:
Nitpick: this function should be `unsafe` as it relies on the `table` holding valid indices.

Reply:
I would say this is very reasonable: the methods that delegate to it could use an unsafe block + relevant safety comment verifying that the table is correctly generated.